scipy-1.0/subpackages.tex

```diff
@@ -73,23 +71,19 @@
     window functions; wavelet computations; peak finding; and spectral
     analysis.  
 \item[\texttt{sparse}]
-    This package includes implementations of several representations of
-    sparse matrices.  It contains two modules, 
-    \texttt{scipy.sparse.linalg} and \texttt{scipy.sparse.csgraph}.
-
-    \texttt{scipy.sparse.linalg} provides a collection of linear algebra
-    routines that work with sparse matrices, including linear equation
-    solvers, eigenvalue decomposition, singular value decomposition
-    and LU factorization.
-
+    This subpackage includes implementations of several representations of
+    sparse matrices.\newline % \texttt{scipy.sparse.linalg} extends past margin
+		\texttt{scipy.sparse.linalg} provides a collection of linear algebra routines that work with
+		sparse matrices, including linear equation solvers, eigenvalue decomposition, singular value
+		decomposition and LU factorization.
     \texttt{scipy.sparse.csgraph} provides a collections of graph algorithms
     for which the graph is represented using a sparse matrix.  Algorithms
     include connected components, shortest path, minimum spanning tree
     and more.
 \item[\texttt{spatial}]
     This subpackage provides spatial data structures and algorithms,
     including the $k$-d tree, Delaunay triangulation, convex hulls and Voronoi
-    diagrams.  The subpackage \texttt{scipy.spatial.distance} provides
+    diagrams. \texttt{scipy.spatial.distance} provides
```

Comment:
This is actually a module, too.

Reply:
I think your decision to remove the specification here is just fine, so I'll mark as resolved.